README.md

```diff
@@ -55,11 +55,37 @@ The application has jasmine tests, which can be accessed at `/specs` when the ap
 [content_schema_examples]: https://github.com/alphagov/finder-frontend/blob/master/lib/govuk_content_schema_examples.rb
 
 ##Making a new finder
-1. If required, add a schema to [alphagov/rummager](http://github.com/alphagov/rummager) describing your document type -- eg
-   [https://github.com/alphagov/rummager/blob/master/config/schema/default/doctypes/cma_case.json](https://github.com/alphagov/rummager/blob/master/config/schema/default/doctypes/cma_case.json)
+1. If required, add a schema to [alphagov/rummager](http://github.com/alphagov/rummager) describing your document type -- [example](https://github.com/alphagov/rummager/blob/master/config/schema/document_types/cma_case.json)
 2. Publish a Finder Content Item to the content store. See the doc for [Finder Content Item](https://github.com/alphagov/finder-frontend/blob/master/docs/finder-content-item.md) for more info.
 3. Ensure your documents are indexed in [alphagov/rummager](http://github.com/alphagov/rummager) correctly.
 
+
+### How to publish a finder?
+
+#### When the publishing app is whitehall
+
+- You will need to create a JSON file in the whitehall repository, under [whitehall/lib/finders][finders-folder]
+- You can base it on one of the existing files in that folder
+- Double-check the filter format and document noun - the filter format is used for rummager to return the data, while the document noun is displayed to the user
+- The default_documents_per_page key can be used to paginate very long finders (see [whitehall/lib/finders/case_studies.json][case-studies] for an example)
+- Running the [finders:publish rake task][rake-task] will publish your new finder to the [publishing-api](https://github.com/alphagov/publishing-api), the the route defined in the JSON will be taken over by finder-frontend
+
+[finders-folder]: https://github.com/alphagov/whitehall/tree/master/lib/finders
+[case-studies]: https://github.com/alphagov/whitehall/blob/master/lib/finders/case_studies.json
+[rake-task]: https://github.com/alphagov/whitehall/blob/master/lib/tasks/publish_finders.rake
+
+#### How can I know the format needed?
```

Comment:
Let's link to the GitHub source, it's good enough and won't change. I'm not sure about using separate GitHub pages for separate projects. The current rummager docs look crappy.